scripts/generators/asciidoc_fields.py

```diff
@@ -76,8 +77,15 @@ def render_field_details_row(field):
     example = ''
     if 'example' in field:
         example = "example: `{}`".format(str(field['example']))
+
+    field_name_with_mf = field['flat_name']
```

Comment:
I think we will need to add some more docs to this code or extract it to documented functions. Not required now).

Reply:
@ruflin Yes, I would like to have the field set pages be expanded this way eventually:

- Title & description of field set at the top (like now)
- More compact table with short definitions only, and field names are links to a more complete section lower in the page
- Perhaps a "field reuse" section like now, or integrate field reuse directly in this summary table described above
- One linkable section per field name, with more complete details. Using headings for those will allow for the linking, and will also allow for additional information that's currently difficult to capture in the table we have not (should it be an array or a single value?, multi-fields details, deprecation, is the field index:false, etc.)